### ISSUE TITLE
Intended use? Service available?

### ISSUE THREAD

**Issue by barry-johnson** [pmq20/enclose-io#24]
Since there is little/no doc here and with just a link to the enclose.io site, I thought maybe this was being offered as service somehow, but the enclose.io site is just informational. Curious if the intention is that users simply download the repo and host their own enclose.io instance or if you are setting this up as a service of some sort. Thanks!

**Comment by pmq20** [comment]
Yes this is offered as a service. The ultimate goal is that user can register their project into Enclose.IO and Enclose.IO will produce stand-alone executables automatically with auto-update enabled.
